src/main.ts

```diff
@@ -367,7 +361,17 @@ export async function activate(context: vscode.ExtensionContext) {
     return {
         getRootFile: () => extension.manager.rootFile,
```

Comment:
It seems that this is duplicating the `rootFile` below. What about removing this and making corresponding changes in LU, specifically in word counter? We may also need to have a general rule of thumb in what structure api should be roughly arranged.

Reply:
The reason that's there is because of these lines in `LU`.
```ts
if (this.extension.workshop.manager.rootFile() === undefined) {
    await this.extension.workshop.manager.findRoot()
}
```
If that's unnecesary I'll just remove that.

Re: api structure, what've I've been going with so far is just mirroring the structure in LW, for example, see above.

Reply:
Yes, please remove. I will handle the word counter changes.

Reply:
done